pulp_puppet_plugins/pulp_puppet/plugins/importers/metadata.py

```diff
@@ -10,33 +10,26 @@
 import tempfile
 
 from pulp.common.compat import json
-from pulp.server.exceptions import InvalidValue
+from pulp.server.exceptions import PulpCodedException
 
 from pulp_puppet.common import constants
+from pulp_puppet.plugins import error_codes
 
 
-class ExtractionException(InvalidValue):
-    """
-    Root exception of all exceptions that can occur while extracting a module's
-    metadata.
-    """
-    def __init__(self, module_filename):
-        InvalidValue.__init__(self, module_filename)
-        self.module_filename = module_filename
-
-
-class MissingModuleFile(ExtractionException):
+class MissingModuleFile(PulpCodedException):
```

Comment:
Is it that the module is missing or the Metadata File? We probably can't find the module because we can't find the metadata so maybe s/MissingModuleFile/MissingMetadataFile/ ?